app/src/main/java/org/fossasia/openevent/fragments/BookmarksFragment.java

```diff
@@ -69,6 +70,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         } catch (ParseException e) {
             Timber.e("Parsing Error Occurred at BookmarksFragment::onCreateView.");
         }
+        if(!bookmarkedIds.isEmpty())
+        {
+            noBookmarks.setVisibility(View.GONE);
```

Comment:
As you can see, when I see code that is not symmetric i feel there would be a bug. 
STR:
1. bookmark a session
2. go to bookmarks page
3. you should see the session, now click on it and un-bookmark it
4. Go back, tell me what you see.

here is what I see.
![image](https://cloud.githubusercontent.com/assets/547235/12812067/7ca96b32-cb6b-11e5-9ab0-2d168d7c37e7.png)